test/test_errors.py

```diff
@@ -85,10 +85,10 @@ def test_handle_result_error(container_factory, rabbit_config):
             # proxy.task() will hang forever because it generates an error
             proxy.task.call_async()
 
-        with eventlet.Timeout(1):
```

Comment:
Travis is variable. Re-running master now is taking 4-5 mins: https://travis-ci.org/nameko/nameko/builds/177243811

There is some small impact here on tear-down times, but I think it is just `0.1` seconds. It becomes apparent in here because we do it a _lot_.

Do we really care that services take slightly longer to stop? I could retrofit the call to `conn.heartbeat_check()` into our lifted method, but I'd rather remove the override. We wouldn't justify _adding_ the override for a small improvement in test-run time.